### ISSUE TITLE
"errorMessage": "Alexa.CreateStateHandler is not a function" with alexa-sdk included?

### ISSUE THREAD

**Issue by ryandeming** [alexa/skill-sample-nodejs-audio-player#17]
Downloaded this repo, created node_modules, added alexa-sdk to it (with node_modules i18next).

Cannot for the life of me figure out how to get this remotely close to working. Haven't touched any of the code in any of the files. Just uploaded them to Lambda and added the ARN to the Alexa skill I created.

Why wouldn't it recognize CreateStateHandler if alexa-sdk is successfully included?

**Comment by rjamison** [comment]
Is it possible you're not recursively zipping up your project?  It could be that "node_modules" is in the wrong location or that they're not included in the zip.

**Comment by ryandeming** [comment]
I downloaded one of the samples off of the Amazon facts demo and structured it the same way. Node_modules is in the root of the zip with the contents of the audioplayer.

I was originally having issues with it not recognizing the alexa sdk, but solved that when I realized I needed to include it in the zip. I believe it's in the right spot now since declaring Alexa isn't throwing an error anymore. 

edit: here's the inside of the zip I'm uploading:
root: http://prntscr.com/dpgoub
root/node_modules: http://prntscr.com/dpgpm6
root/node_modules/alexa-sdk: http://prntscr.com/dpgpsz

edit2:
not sure why, but it started working this morning. didn't change anything.
